app/modules/front-door-premium/README.md

```diff
@@ -13,7 +13,7 @@ This component contains the Azure Front Door and WAF resources. The resources in
 
 | Name | Version |
 |------|---------|
-| <a name="provider_azurerm"></a> [azurerm](#provider\_azurerm) | 3.116.0 |
+| <a name="provider_azurerm"></a> [azurerm](#provider\_azurerm) | ~> 3.0 |
```

Comment:
I don't know if this is an issue?

Reply:
Shouldn't be, just states accept any version of azurerm 3.x.x

Reply:
reckon we approve for now and see what they say afterwards